dao/test/end_vote.test.mligo

```diff
@@ -0,0 +1,112 @@
+#import "tezos-ligo-fa2/test/helpers/list.mligo" "List_helper"
+#import "./helpers/dao.mligo" "DAO_helper"
+#import "./helpers/time.mligo" "Time_helper"
+#import "./helpers/log.mligo" "Log"
+#import "./helpers/assert.mligo" "Assert"
+#import "./bootstrap/bootstrap.mligo" "Bootstrap"
+#import "../src/main.mligo" "DAO"
+
+let () = Log.describe("[End_vote] test suite")
+
+(* Boostrapping of the test environment, *)
+let init_tok_amount = 33n
+let bootstrap () = Bootstrap.boot(init_tok_amount)
+
+(* Successful end_vote with proposal accepted *)
+let test_success_prop_accepted = 
+    let (tok, dao, sender_) = bootstrap() in
+
+    (* 
+        in order for this test to be succesful,
+        each token owner locks 25 tokens, 
+        so that quorum is reached
+    *)
+    let amount_ = 25n in
+    let () = DAO_helper.batch_lock(tok.owners, amount_, dao.contr) in
+    (* restore back bootstrap sender *)
+    let () = Test.set_source sender_ in
+    let () = DAO_helper.propose_success(DAO_helper.dummy_proposal, dao.contr) in
+    let dao_storage = Test.get_storage dao.taddr in
+
+    (* 
+        all voters vote the same, so that super_majority is reached
+    *)
+    let () = DAO_helper.batch_vote(tok.owners, true, dao.contr) in
+    let () = Time_helper.advance(dao_storage.config.start_delay) in
+    (* restore back bootstrap sender *)
+    let () = Test.set_source sender_ in
+    let () = DAO_helper.end_vote_success(dao.contr) in
```

Comment:
This looks similar the `Suite_helper.make_proposal_success` with lambda as `None`

Reply:
yes, thank you, I refactored a little